### PR TITLE
News.md updated for 2.5.0

### PR DESCRIPTION
**Pull request by vijay-hirani**
NEWS.md updated for  2.5.0

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+* implement path monitoring profiles for outgoing (active) paths
+  * different echo interval configurable depending whether path is 
+    active (has contexts), idle (no contexts) or is down
+  * Monitoring of max duration of Idle and Down paths
+* rewrite state logic of gtp path
+* make path management defaults configurable
+* bump ergw_aaa dependency to 3.5.0 to support state stats and 
+  diameter avp filters
+* bump prometheus_diameter_collector to report plain error counters
+* fix: start echo sending after getting a response or timeout
```
Comment (RoadRunnr) [inline NEWS.md:15]: this is part of that path monitoring and doesn't need a seperate point in the NEWS update

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+* Add Termination-Cause to accounting stop
```
Comment (RoadRunnr) [inline NEWS.md:16]: that is an ergw internal details that needs proper support in ergw_aaa to have any effect. The releated ergw_aaa change has not been merged, as it is incomplete.

**Comment by vijay-hirani** [comment]
NEWS.md updates with links to corresponding features. Internal changes removed from the NEWS.md

**Comment by vijay-hirani** [comment]
I have taken into account the comments from Mathias and Andreas. 

If there are no further comments then can I request Andreas to merge the NEWS.md into master.  
So that I can up the tag to 2.5.0.  Thanks

**Comment by vijay-hirani** [comment]
@RoadRunnr 
rebased and updated the NEWS.md file with today's date.   Please  merge if okay.

File: NEWS.md [hunk 3]
```diff
@@ -1,6 +1,15 @@
 erGW - 3GPP GGSN and PDN-GW in Erlang
 =====================================
 
+Version 2.5.0 - 04 August 2020
+----------------------------
+* implement path monitoring profiles for outgoing (active) paths
+* bump ergw_aaa dependency to 3.5.0 to support  
+  [state stats](https://github.com/travelping/ergw_aaa/blob/master/doc/aaa_session_metrics.md) and
+  diameter [avp filters](https://github.com/travelping/ergw_aaa/blob/master/doc/diameter_avp_filter.md)
```
Comment (RoadRunnr) [inline NEWS.md:9]: can you change the links to point to the actual tag, e.g.:

https://github.com/travelping/ergw_aaa/blob/3.5.0/doc/diameter_avp_filter.md

**Pull request merged**